pkgs/development/ocaml-modules/mtime/default.nix

```diff
@@ -1,37 +1,25 @@
-{ stdenv, lib, fetchurl, ocaml, findlib, ocamlbuild, topkg, js_of_ocaml
-, jsooSupport ? lib.versionAtLeast ocaml.version "4.03"
-}:
+{ stdenv, lib, fetchurl, ocaml, findlib, ocamlbuild, topkg }:
 
 with lib;
 
-let param =
-  if versionAtLeast ocaml.version "4.03"
-  then {
-    version = "1.2.0";
-    sha256 = "0zm1jvqkz3ghznfsm3bbv9q2zinp9grggdf7k9phjazjvny68xb8";
-  } else {
-    version = "0.8.4";
-    sha256 = "1adm8sc3lkjly99hyi5gqnxas748k7h62ljgn8x423nkn8gyp8dh";
-  };
-in
+throwIfNot (versionAtLeast ocaml.version "4.08")
+  "mtime is not available for OCaml ${ocaml.version}"
 
-stdenv.mkDerivation {
+stdenv.mkDerivation rec {
   pname = "ocaml${ocaml.version}-mtime";
-  inherit (param) version;
+  version = "1.4.0";
 
   src = fetchurl {
-    url = "https://erratique.ch/software/mtime/releases/mtime-${param.version}.tbz";
-    inherit (param) sha256;
+    url = "https://erratique.ch/software/mtime/releases/mtime-${version}.tbz";
+    sha256 = "sha256-VQyYEk8+57Yq8SUuYossaQUHZKqemHDJtf4LK8qjxvc=";
```

Comment:
Note the preferred attribute is apparently now `hash` rather than `sha256`.

Reply:
Good to know. Thanks.